app/models/grant.rb

```diff
@@ -1,28 +1,30 @@
 class Grant < ActiveRecord::Base
   has_many :grants_voters
+  has_many :grant_submissions
 
-  validates :name, :presence => true
-  validates :max_funding_dollars, :presence => true, :numericality => {:greater_than => 0, :only_integer => true}
-  validates :submit_start, :presence => true
-  validates :submit_end, :presence => true
-  validates :vote_start, :presence => true
-  validates :vote_end, :presence => true
-  validates :meeting_one, :presence => true
-  validates :meeting_two, :presence => true
+  validates :name, presence: true
+  validates :max_funding_dollars, presence: true, numericality: { greater_than: 0, only_integer: true}
+  validates :submit_start,  presence: true
```

Comment:
cools. It was hard from all the various tutorials and code snippets to figure out which syntax was old and which was new.  It also feels like ruby/rails has undergone a lot of changes over time with the styles changing pretty significantly!